about/structure.md

```diff
@@ -0,0 +1,60 @@
+# Structure and governance of 2i2c
+
+2i2c is an initiative dedicated to supporting open infrastructure in interactive computing for research and education, as well as the open source communities that underlie this infrastructure. Its host organization is {ref}`structure:icsi`.
```

Comment:
```suggestion
2i2c is an organization dedicated to supporting open infrastructure in interactive computing for research and education, as well as the open source communities that underlie this infrastructure. Its host organization is {ref}`structure:icsi`.
```

Reply:
I was using initiative to prevent people from thinking we are a separate 501c3. Do you not share this concern?

Reply:
The problem with "initiative" is that it sounds like it has a fixed time horizon. What happens when the initiative ends?

I see no problem with using "organization" in this way but don't feel strongly.